src/components/Data/itemsStorage.ts

```diff
@@ -0,0 +1,7 @@
+export function getLocalStorageItem(item: string) {
```

Comment:
Do we really need a function that encapsulates one line of code, consisting of calling another function with the same name, minus a point and with the same amount of parameters?
I don't think so... Here, you are just adding one layer of `import` statements without introducing any abstraction nor currying.
It is not bad to directly calling the Local Storage API from within the components 😄

Reply:
We talked about this with you and Nico and the conclusion was that currently we are using the local storage but maybe one day it will be a database (or anything else) and that encapsulation will be useful. So in order to make it easier in the future we decided to do it like that. If you think it's still unnecessary I can change it!

Reply:
Oook, now I remember, thanks for recalling. It is, indeed, a little bit overkill, but let's encapsulate all the calls to Local Storage, it's not a bad idea. But, if you want to encapsulate, you should avoid using the word "LocalStorage" in the name of the wrapper function 😅 Think about the day we switch to SQLLite: we'd have to rename all the calls to `getLocalStorageItem` to `getSQLLiteItem`, which is still a bad practice. When encapsulating, use an abstract name, like `getPersistedItem` and leave to the implementation the meaning of `Persisted`. Sounds clear?